chain/vm/runtime.go

```diff
@@ -44,6 +44,8 @@ type Runtime struct {
 	origin address.Address
 
 	internalExecutions []*ExecutionResult
+	// tracks internal call depth, starts at 0
+	callDepth uint64
```

Comment:
Is the depth what we want here? I'm not sure.
What if method A calls method B which creates an actor and then the method A calls method C which also creates an actor.
With callDepth both B and C would have the same depth so resulting addr would be the same.

Reply:
Ooh, good catch, thanks! This has been fixed to use `callDepth + len(internalExecutions)`.